features/ProductNews.feature

```diff
@@ -0,0 +1,46 @@
+@owner:derek
+@author:lilla
+
+Feature: Docs Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario: There are two filter options at the top of the news column to filter based on tags and dates
+        Given a user wants to filter the news feed
+        Then they can select from the following dropdown filter items for type
+
+            | postType         |
+            | All Posts        |
+            | Feature Releases |
+            | Monthly Updates  |
+            | Product Updates  |
+
+        And they can select from the following dropdown items for dates
```

Comment:
See above, you may even just want to break these out into two different specs. Product news updates are filterable by type and Product news updates are filterable by date